src/Moryx.Runtime.Endpoints/Databases/Endpoint/DatabaseController.cs

```diff
@@ -186,23 +187,7 @@ public ActionResult<DatabaseUpdateSummary> MigrateDatabaseModel(string targetMod
                 throw new ArgumentException("Config values are not valid", nameof(configModel));
```

Comment:
Should be returning NotFound instead?

Reply:
Good point, but I didn't intent do change the endpoints in this MR. I just fixed it by accident trying to get it to compile again.